tests/playwright/src/model/pages/create-machine-page.ts

```diff
@@ -0,0 +1,137 @@
+/**********************************************************************
```

Comment:
Recently as part of creating a new podman machine test I've noticed that the `podman-onboarding-page.ts` contains exactly the same content as this new page would, the issue is that the onboarding page is simply misnamed, initially it was created a POM class to be used in the onboarding workflow for podman machine tests, but what it actually models is that podman resource creation machine, just like this one, it would make sense to rename the old page and refactor it instead of creating a new page with the same content but with a new name.

In my test I actually repurposed that page and used it for creating a podman machine.

Reply:
They are not the same page, it just contains the same properties information form (name, checkboxes). I could add the properties form into workbench and then have it repurposed in both pages, but I'd like to keep the pages separate.

Reply:
The pages are the same, the issue is that the page is reused in the app in different contexts, but the creation page is identical, we should not have multiple POM pages that address the same app page, yes there are some differences due to how the onboarding workflow works, but one can think of those differences as extra pages also. Maintaining to POMs for the same page is not scalable and defeats the purpose of the POM model.